docs/getting-started/3-device-examples/2-compile-example-code/3-esp-idf/README.md

```diff
@@ -1,23 +1,22 @@
 ---
 id: overview
-title: ESP32 Overview
+title: ESP-IDF with Golioth
 ---
 
-## What is the ESP32?
+## What is the ESP-IDF?
 
-The ESP32 is a Wi-Fi and Bluetooth combination chip from [Espressif Systems](https://espressif.com/). There are multiple versions of the chip, with the most recent versions running in different configurations of core types, up to and including a RISC V core (on the ESP32-C3).
+The Espressif IoT Development Framework
+([ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/)) is a
+FreeRTOS-based platform developed and maintained by [Espressif
+Systems](https://espressif.com/) for their line of ESP32 microcontrollers. The
+Golioth Firmware SDK can be included in your projects as an ESP-IDF component.
 
-The Espressif team has built-in support for Zephyr. This, in combination with the low cost and wide availability of the chips, has led to the ESP32 being one of the first platforms that Golioth has chosen to support.
+## Using ESP32 with Golioth
 
-## Terminology
+The ESP32 is a Wi-Fi and Bluetooth combination chip. There are multiple versions
+of the chip, with the most recent versions running in different configurations
+of core types, up to and including a RISC V core (on the ESP32-C3).
```

Comment:
"up to and including" sounds a little weird to me, like we're ranking architectures. This is existing copy though, so I'm fine with leaving this as is.

Reply:
```
of the chip, including the ESP32, ESP32-S2, and ESP32-S3 based on the Xtensa
core and the ESP32-C3 which uses the RISC-V core.
```

You're right that this is wonky. I'll update to list parts and cores. It's a good place to indicate our breadth of support.